Formula/coreutils.rb

```diff
@@ -57,6 +57,7 @@ def install
       --program-prefix=g
       --with-gmp
       --without-selinux
+      --enable-install-program=hostname
```

Comment:
Thanks for the PR.

Would you know why this isn't enabled by default? We usually get a little leery about deviating from the default build configurations, since the devs typically set those to the most sensible configuration. Do other package managers enable `hostname` in their build of `coreutils`?

Reply:
Hi @carlocab,  I'll double check some other package managers and see what they are doing. In the case of this package, these utilities are installed with a prefix, `g`, so this would be installed as `ghostname` and not override any other `hostname` packages (at least not by default).

The GNU Coreutils docs provides some insight into why it's not enabled by default:
```
hostname is not installed by default, and other packages also supply a hostname command, so portable scripts should not rely on its existence or on the exact behavior documented above.
```

Reply:
I checked Fedora and Debian and by default they ship this `hostname` package (and do not install hostname from coreutils):
http://packages.qa.debian.org/h/hostname.html

Reply:
Yes, I think we'll want to follow Fedora and Debian here I think.